test/fixtures/valid-geojson.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
Thanks for the feedback @springmeyer! I fixed the formatting of the test fixture, deleted the `id` property, and made all the feature properties unique. All the properties were successfully extracted.